src/crimson/os/seastore/onode_manager/staged-fltree/node.cc

```diff
@@ -319,22 +314,22 @@ node_future<Node::search_result_t> Node::lower_bound(
   );
 }
 
-node_future<std::pair<Ref<tree_cursor_t>, bool>> Node::insert(
+eagain_future<std::pair<Ref<tree_cursor_t>, bool>> Node::insert(
     context_t c, const key_hobj_t& key, value_config_t vconf)
 {
   return seastar::do_with(
     MatchHistory(), [this, c, &key, vconf](auto& history) {
       return lower_bound_tracked(c, key, history
       ).safe_then([c, &key, vconf, &history](auto result) {
         if (result.match() == MatchKindBS::EQ) {
-          return node_ertr::make_ready_future<std::pair<Ref<tree_cursor_t>, bool>>(
+          return eagain_ertr::make_ready_future<std::pair<Ref<tree_cursor_t>, bool>>(
               std::make_pair(result.p_cursor, false));
         } else {
           auto leaf_node = result.p_cursor->get_leaf_node();
           return leaf_node->insert_value(
               c, key, vconf, result.p_cursor->get_position(), history, result.mstat
           ).safe_then([](auto p_cursor) {
-            return node_ertr::make_ready_future<std::pair<Ref<tree_cursor_t>, bool>>(
+            return seastar::make_ready_future<std::pair<Ref<tree_cursor_t>, bool>>(
```

Comment:
Because this return doesn't contain any error, another example see https://github.com/ceph/ceph/blob/12fd0f1eeadf8ea64aaec59a02fe1af40598f0a4/src/crimson/os/seastore/seastore.cc#L209